README.adoc

```diff
@@ -1,22 +1,177 @@
+
+
+
+Il s’agit d’une fonctionnalité proposée car à ce jour, cette dernière n’a pas été traitée.
+Ainsi, nous ne faisons pas la différence entre les personnes qui créent des questions (Auteurs) et celles qui utilisent les questions pour s'entraîner.
+
+
+
+
+*Khaled*
+
+
+
+* Niveau de difficulté : Pour les QCM, les réponses proposées auront un niveau de difficulté. Les réponses fausses aurons deux compteurs, un pour compter le nombre de participation aux examens, et l'autre pour compter le nombre de fois qu'elle a été sélectionnée par les candidat. le ratio [nombre participation / nombre de sélection ] donne le degré de corrélation entre cette réponse et la bonne réponse, et donc si le ratio est proche de 1 la réponse aura une forte probabilité d'être choisi. Si cette réponse apparaît dans l'examen, on dit que la question est difficile.
+
+
+
+*Remarque*
+au début toute les réponse sont proposées au même niveau de corrélation, après un certain nombre d'utilisation
+
+*Mohamed*
+
+
+* Question Xml : Lire et écrire une question en XML, et les relations de votre modèle. Chaque relation peut être représentée comme un ensemble de triplets associé à une personne(1)
+
+* Faire les différentes conversion en Xml (QuestionText)
+
+
+* Servlet Delet:   Ajouter une servlet qui permet aux autor de supprimer les questions de leur choix dans la BDD avec un cookie qui mémorise id personne. la suppression se fait soit avec un id questions, suppression total,par un id author... Dans la servlet elle permet aussi au enseignants de supprimer les examens qui seront créés (1)
+
+
+*Valentin*
+
+
+
+* Gestion des questions : Un utilisateur peut indiquer sur chaque question: un coefficient pour chaque question, la matière, le chapitre associé et le niveau recommandé
+Création de regroupement de questions appelé Examen selon la matière, le chapitre ou le niveau recommandé (de CP à Terminale)
+* Les examens seront exportables en fichier .pdf
+* Création de la page Author reprenant les noms des auteurs des questions et divers statistiques
+* Création de la page About Us qui reprendra les tâches et fonctionnalités developpés par chacun des membres du groupe
+* Création de la page Contact Us avec les coordonnées nécessaires de chaque membre ainsi que les liens utiles
+
+*Katia*
+
+
+
+* ServletsModify: ChangeResponseTypeTF(id orig, new author id): new id et ChangeResponseTypeYN(id orig, new author id). Change la réponse à une question en créant une nouvelle question qui a un autre type de réponse (c-à-d Y/N ou T/F) et en renvoyant son nouvel id. 
+
+* ChangeQuestionLanguage(id orig, new author id, new language): new id ( modifier les classes existantes par rapport à ce qu’il a été demandé dans l’énoncé, récupération des paramètres soumis dans le formulaire HTML)
+
+*Alexandre*
+
+
+
+* Online : Faire tourner le serveur en ligne grâce au service d’IBM. (0,5)
+* UseDB : Modifier les servlets pour qu’ils écrivent dans et lisent la BD. (1)
+
+
+= Deuxième itération
+

```

Comment:
Vous pouvez déplacer les fct concernant le GUI ici, si vous le désirez. On verra plus tard exactement qui fait quoi pour itération 2 et 3.